install error during vet step

# What did you do? (required. The issue will be **closed** when not provided.)
make install

# What did you expect to happen?
installation 

# What happened instead?
installation failed during vet step 

* Current Output

```
GO111MODULE=off go get -u golang.org/x/lint/golint
golint github.com/future-architect/vuls github.com/future-architect/vuls/alert github.com/future-architect/vuls/cache github.com/future-architect/vuls/commands github.com/future-architect/vuls/config github.com/future-architect/vuls/contrib/owasp-dependency-check/parser github.com/future-architect/vuls/cwe github.com/future-architect/vuls/errof github.com/future-architect/vuls/exploit github.com/future-architect/vuls/github github.com/future-architect/vuls/gost github.com/future-architect/vuls/libmanager github.com/future-architect/vuls/models github.com/future-architect/vuls/oval github.com/future-architect/vuls/report github.com/future-architect/vuls/scan github.com/future-architect/vuls/server github.com/future-architect/vuls/util github.com/future-architect/vuls/wordpress
echo github.com/future-architect/vuls github.com/future-architect/vuls/alert github.com/future-architect/vuls/cache github.com/future-architect/vuls/commands github.com/future-architect/vuls/config github.com/future-architect/vuls/contrib/owasp-dependency-check/parser github.com/future-architect/vuls/cwe github.com/future-architect/vuls/errof github.com/future-architect/vuls/exploit github.com/future-architect/vuls/github github.com/future-architect/vuls/gost github.com/future-architect/vuls/libmanager github.com/future-architect/vuls/models github.com/future-architect/vuls/oval github.com/future-architect/vuls/report github.com/future-architect/vuls/scan github.com/future-architect/vuls/server github.com/future-architect/vuls/util github.com/future-architect/vuls/wordpress | xargs env GO111MODULE=on go vet || exit;
go: finding github.com/kotakanbe/goval-dictionary v0.1.3-0.20190613053258-8b98657de17d
go: github.com/kotakanbe/goval-dictionary@v0.1.3-0.20190613053258-8b98657de17d: unknown revision 8b98657de17d
go: error loading module requirements
GNUmakefile:42: recipe for target 'vet' failed
make: *** [vet] Error 123
```

Please re-run the command using ```-debug``` and provide the output below.

```
GNU Make 4.1
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2014 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Reading makefiles...
Updating goal targets....
 File 'install' does not exist.
   File 'pretest' does not exist.
     File 'lint' does not exist.
    Must remake target 'lint'.
GO111MODULE=off go get -u golang.org/x/lint/golint
golint github.com/future-architect/vuls github.com/future-architect/vuls/alert github.com/future-architect/vuls/cache github.com/future-architect/vuls/commands github.com/future-architect/vuls/config github.com/future-architect/vuls/contrib/owasp-dependency-check/parser github.com/future-architect/vuls/cwe github.com/future-architect/vuls/errof github.com/future-architect/vuls/exploit github.com/future-architect/vuls/github github.com/future-architect/vuls/gost github.com/future-architect/vuls/libmanager github.com/future-architect/vuls/models github.com/future-architect/vuls/oval github.com/future-architect/vuls/report github.com/future-architect/vuls/scan github.com/future-architect/vuls/server github.com/future-architect/vuls/util github.com/future-architect/vuls/wordpress
    Successfully remade target file 'lint'.
     File 'vet' does not exist.
    Must remake target 'vet'.
echo github.com/future-architect/vuls github.com/future-architect/vuls/alert github.com/future-architect/vuls/cache github.com/future-architect/vuls/commands github.com/future-architect/vuls/config github.com/future-architect/vuls/contrib/owasp-dependency-check/parser github.com/future-architect/vuls/cwe github.com/future-architect/vuls/errof github.com/future-architect/vuls/exploit github.com/future-architect/vuls/github github.com/future-architect/vuls/gost github.com/future-architect/vuls/libmanager github.com/future-architect/vuls/models github.com/future-architect/vuls/oval github.com/future-architect/vuls/report github.com/future-architect/vuls/scan github.com/future-architect/vuls/server github.com/future-architect/vuls/util github.com/future-architect/vuls/wordpress | xargs env GO111MODULE=on go vet || exit;
go: finding github.com/kotakanbe/goval-dictionary v0.1.3-0.20190613053258-8b98657de17d
go: github.com/kotakanbe/goval-dictionary@v0.1.3-0.20190613053258-8b98657de17d: unknown revision 8b98657de17d
go: error loading module requirements
GNUmakefile:42: recipe for target 'vet' failed
make: *** [vet] Error 123
```

# Steps to reproduce the behaviour
1. clone repository
2. execute make install

# Configuration (**MUST** fill this out):

* Go version (`go version`): go version go1.12.5 linux/amd64

* Go environment (`go env`):
```
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/ubuntu/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/opt/ptools/scan/go"
GOPROXY=""
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build837785299=/tmp/go-build -gno-record-gcc-switches"
```


* Vuls environment:

Hash : b81f640

To check the commit hash of HEAD
$ vuls -v

or

$ cd $GOPATH/src/github.com/future-architect/vuls 
$ git rev-parse --short HEAD 

* config.toml:

* command: make install

--- a/go.mod
+++ b/go.mod
@@ -51,7 +51,7 @@ require (
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/kotakanbe/go-cve-dictionary v0.0.0-20190327053454-5fe52611f0b8
 	github.com/kotakanbe/go-pingscanner v0.1.0
-	github.com/kotakanbe/goval-dictionary v0.1.3-0.20190613053258-8b98657de17d
+	github.com/kotakanbe/goval-dictionary v0.1.3-0.20190613053258-078b163b76ec
 	github.com/kotakanbe/logrus-prefixed-formatter v0.0.0-20180123152602-928f7356cb96
 	github.com/kr/pty v1.1.5 // indirect
 	github.com/labstack/echo v3.3.10+incompatible // indirect

